cfme/tests/test_performance.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+import pytest
+
+from cfme.base.ui import navigate_to
+from cfme.services.myservice import MyService
+from cfme.tests.test_db_migrate import download_and_migrate_db
+from cfme.utils.conf import cfme_data
+
+
+class TimedCall(object):
```

Comment:
Did you consider using `timeit` here?

Reply:
Sure, but it accepts only strings of python code. So it is cumbersome to use here.

```
In [1]: o = object()

In [2]: import timeit

In [3]: timeit.timeit("str(o)")
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-3-d7f20afbb7aa> in <module>()
----> 1 timeit.timeit("str(o)")

/usr/lib64/python2.7/timeit.pyc in timeit(stmt, setup, timer, number)
    235            number=default_number):
    236     """Convenience function to create Timer object and call timeit method."""
--> 237     return Timer(stmt, setup, timer).timeit(number)
    238 
    239 def repeat(stmt="pass", setup="pass", timer=default_timer,

/usr/lib64/python2.7/timeit.pyc in timeit(self, number)
    200         gc.disable()
    201         try:
--> 202             timing = self.inner(it, self.timer)
    203         finally:
    204             if gcold:

/usr/lib64/python2.7/timeit.pyc in inner(_it, _timer)

NameError: global name 'o' is not defined
```

Reply:
Well I tried and found it is undocumented feature, that it accepts also callable. I wonder is it good idea to use it.